src/app/components/ATIAnalytics/index.test.tsx

```diff
@@ -306,6 +306,7 @@ describe('ATI Analytics Container', () => {
         lng: 'en-US',
         x2: '[responsive]',
         x3: '[news]',
+        x4: '[en-gb]',
```

Comment:
Is this OK / as expected?

Reply:
Seems ok. This is taken from a production News AMP article:
<img width="468" alt="Screenshot 2024-08-14 at 13 21 01" src="https://github.com/user-attachments/assets/2db359c6-7947-42e0-9d40-4a98d6eb6a88">

Reply:
This test has `x2` as `responsive` which is canonical, but we won't render canonical News articles.

Reply:
Ya, we can render canonical news on local, but not Test / Live 😄